### PR TITLE
Add python_requires to help pip

### PR DESCRIPTION
**Pull request by hugovk**
Suggested in https://github.com/user-cont/colin/issues/202#issuecomment-589707226, helps pip install the best version for the user's running Python.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
                                            "rulesets/fedora.json"]),
                 ("share/bash-completion/completions/", ["bash-complete/colin"])],
     license='GPLv3+',
+    python_requires=">=3.5",
```
Comment (ericwb) [inline setup.py]:
```suggestion
    python_requires=">=3.6",
```

Comment (lslebodn) [reply setup.py]: I do not have preferences about 3.5 vs 3.6
 but +1 for for adding requirements >3.x

Comment (hugovk) [reply setup.py]: Good spot, this should be 3.6. I'll update the classifiers too.

**Comment by lslebodn** [comment]
ok to test

**Pull request merged**